School/book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader rental
+
+  def initalize(title, author)
+    @title = title
+    @author = author
+    @rental = rental[]
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
```

Comment:
- [ ] As requested by the previous code reviewer,  book itself and the person should be sent as a parameter to create the new rental achieving the 'has-many' association. Your `add_rental` method is still missing person and book parameters. Please kindly fix that.